pkg/snapshot/store.go

```diff
@@ -158,7 +149,15 @@ func ConfigureStore(ctx context.Context, options ConfigureStoreOptions) (*types.
 	return updatedStore, nil
 }
 
-func updateExistingStore(ctx context.Context, clientset kubernetes.Interface, store *types.Store, options ConfigureStoreOptions) (*types.Store, bool, error) {
+func buildNewStore(ctx context.Context, clientset kubernetes.Interface, options ConfigureStoreOptions) (*types.Store, bool, error) {
+	store, err := GetGlobalStore(ctx, options.KotsadmNamespace, nil)
```

Comment:
`GetGlobalStore` needs a k8s client, so the unit tests may fail unless we make it accept one and pass it through here.  Tests will likely need to be updated too if we go that route.